shuup/front/basket/command_dispatcher.py

```diff
@@ -73,8 +74,9 @@ def handle(self, command, kwargs=None):
         except (Problem, ValidationError) as exc:
             if not self.ajax:
                 raise
+            msg = exc.message if hasattr(exc, "message") else exc
```

Comment:
Exception.message is deprecated, proper way to get the message is by casting to string:
```python
msg = '{}'.format(exc)
```
(Note: should have `from __future__ import unicode_literals` too)